Sources/SWBCore/Specs/Tools/LinkerTools.swift

```diff
@@ -559,7 +559,7 @@ public final class LdLinkerSpec : GenericLinkerSpec, SpecIdentifierType, @unchec
         // Add flags to emit SDK imports info.
         let sdkImportsInfoFile = cbc.scope.evaluate(BuiltinMacros.LD_SDK_IMPORTS_FILE)
         let supportsSDKImportsFeature = (try? optionContext?.toolVersion >= .init("1164")) == true
-        let usesLDClassic = commandLine.contains("-r") || commandLine.contains("-ld_classic") || cbc.scope.evaluate(BuiltinMacros.CURRENT_ARCH) == "armv7k"
+        let usesLDClassic = commandLine.contains("-r") || commandLine.contains("-Wl,-r") || commandLine.contains("-ld_classic") || commandLine.contains("-Wl,-ld_classic") || cbc.scope.evaluate(BuiltinMacros.CURRENT_ARCH) == "armv7k"
```

Comment:
While writing a comment for this, I found `enumerateLinkerCommandLine` and I guess that's what I should be using here.